sdk/examples/package.json

```diff
@@ -32,10 +32,10 @@
     "tsx": "^4.7.2"
   },
   "dependencies": {
-    "@wormhole-foundation/sdk": "^2.4.0",
+    "@wormhole-foundation/sdk": "^3.4.3",
     "@wormhole-foundation/sdk-definitions-ntt": "1.0.0",
```

Comment:
We should also make sure all the 1.0.0 and 1.0.6 versions of "@wormhole-foundation/sdk are also updated to 2.0.1 since that will reduce mysten/sui errors too